components/expenses/ExpensePayeeDetails.js

```diff
@@ -232,8 +232,8 @@ ExpensePayeeDetails.propTypes = {
   /** Disable border and paiding in styled card, usefull for modals */
   borderless: PropTypes.bool,
   collective: PropTypes.shape({
-    isApproved: PropTypes.bool,
-  }),
+    isApproved: PropTypes.bool.isRequired,
+  }).isRequired,
```

Comment:
I think it's false to say `collective` is required because it's not if `isLoading=true`